pkg/security/tests/bpf_test.go

```diff
@@ -22,11 +22,11 @@ func TestBPFEventLoad(t *testing.T) {
 	ruleDefs := []*rules.RuleDefinition{
 		{
 			ID:         "test_prog_load",
-			Expression: `bpf.cmd == BPF_PROG_LOAD && process.file.name == "syscall_go_tester"`,
+			Expression: `bpf.cmd == BPF_PROG_LOAD && bpf.prog.name == "kprobe_vfs_open" && process.file.name == "syscall_go_tester"`,
 		},
 	}
 
-	test, err := newTestModule(t, nil, ruleDefs, testOpts{})
+	test, err := newTestModule(t, nil, ruleDefs, testOpts{enableRuntimeCompiledConstants: true})
```

Comment:
Why do we run those test with runtime compiled constants ? I think we should stick to defaults so that we map more clearly to user environments

Reply:
Good catch ! this was a leftover from the time we didn't have the fallback offsets for bpf 😄